.github/workflows/pip_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I would not be happy to move such a simple script to a separate repo. I did not know you could embed a composite action withing the same repo though. That sounds like a great idea. Particularly since this script really needs to include:

```
      - name: Upload lockfiles
        uses: actions/upload-artifact@v3
        with:
          name: lockfiles
          path: lockfiles
```

Reply:
Switched to using a composite action and it made things nice and tidy. Thanks for the tip.